src/style.css

```diff
@@ -0,0 +1,24 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+.main_block {
+  margin: 0;
+  font-size: 17px;
+  font-weight: bold;
+}
+
+p {
+  margin: 0;
+}
+
+.blocks {
+  font-size: 15px;
+  font-weight: 400;
+  margin: 1em 0;
+}
+
+.artickle17to30 {
```

Comment:
Really bad name for the class. You should name it more semantically. Simply `.link` or just use `a` selector (you have only one link (`a` - tag) in your code